haiku-apps/bescreencapture/bescreencapture-2.1.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="A screen recorder utility"
+DESCRIPTION="BeScreenCapture is a screen recorder utility for Haiku.
+It allows you to record what happens on your screen, then save it \
+to any media format supported in Haiku.
+BeScreenCapture can record either the entire screen, or just a section you \
+select."
+
+SUMMARY_inputfilter="Shortcut handler for BeScreenCapture"
+DESCRIPTION_inputfilter="Input Server Addon for BeScreenCapture. Allows the \
+user to launch BeScreenCapture and start/stop recording using a keyboard \
+combination (CTRL-COMMAND-SHIFT + R)."
+
+HOMEPAGE="https://github.com/jackburton79/bescreencapture"
+SOURCE_URI="https://github.com/jackburton79/bescreencapture/archive/v2.1.tar.gz"
+CHECKSUM_SHA256="5b3ff575fcfe7519556fcd3d9487ef5491ead656f1cffd79b78dfcb3a52332f8"
+SOURCE_FILENAME="bescreencapture-2.1.tar.gz"
+LICENSE="
+	BSD (3-clause)
+	MIT
+	"
+COPYRIGHT="2014-2016 Stefano Ceccherini"
+REVISION="1"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+
+PROVIDES="
+	bescreencapture = $portVersion
+	app:BeScreenCapture = $portVersion
+	"
+
```

Comment:
Yep, I was also thinking about the ordering of the _devel subpackages.
But I just noticed that the recipes for git are using the second style, so maybe _devel is a special case.
So I guess the recommended ordering for this recipe might be:

PROVIDES
PROVIDES_inputfilter

REQUIRES
REQUIRES_inputfilter